### PR TITLE
Remove deprecated MXBean APIs & internal methods in Java 19 and 20

### PR DESCRIPTION
**Pull request by pshipton**
Removed in 19:
ExtendedMemoryMXBeanImpl.getTotalPhysicalMemory()
OperatingSystemMXBean.getTotalPhysicalMemory(), getProcessCpuTimeByNS(), getProcessVirtualMemorySize()
RuntimeMXBean.getProcessID()

Removed in 20:
ExtendedMemoryMXBeanImpl.getTotalPhysicalMemorySize(), getUsedPhysicalMemory()

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins compile win jdknext

**Comment by pshipton** [comment]
jenkins test sanity,extended amac jdknext

**Comment by pshipton** [comment]
https://openj9-jenkins.osuosl.org/job/PullRequest-OpenJ9/2172/

**Comment by pshipton** [comment]
jenkins test sanity,extended amac jdknext

**Comment by pshipton** [comment]
jenkins compile win jdknext

**Comment by pshipton** [comment]
https://openj9-jenkins.osuosl.org/job/PullRequest-OpenJ9/2175/

**Comment by pshipton** [comment]
Testing via grinders:
8: https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/822
19: https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/823

File: runtime/jcl/uma/se6_vm-side_natives_exports.xml [hunk 1]
```diff
@@ -171,6 +171,8 @@
 	<export name="Java_com_ibm_java_lang_management_internal_RuntimeMXBeanImpl_getUptimeImpl" />
 	<export name="Java_com_ibm_java_lang_management_internal_RuntimeMXBeanImpl_isBootClassPathSupportedImpl" />
 	<export name="Java_com_ibm_lang_management_internal_ExtendedRuntimeMXBeanImpl_getProcessIDImpl" />
+		<exclude-if condition="spec.java19" />
+	</export>
```
Comment (keithc-ca) [inline runtime/jcl/uma/se6_vm-side_natives_exports.xml:175]: This is not legal XML: the `<export ... />` element should end with `>`:
```
	<export name="Java_com_ibm_lang_management_internal_ExtendedRuntimeMXBeanImpl_getProcessIDImpl">
		<exclude-if condition="spec.java19" />
	</export>
```

File: jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedMemoryMXBeanImpl.java [hunk 2]
```diff
@@ -91,6 +91,7 @@ protected void startNotificationThread() {
 		}
 	}
 
+/*[IF JAVA_SPEC_VERSION < 19]*/
 	/**
 	 * Deprecated. Use getTotalPhysicalMemorySize().
```
Comment (keithc-ca) [inline jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedMemoryMXBeanImpl.java:96]: This should be updated to advise use of something that isn't going away in Java 20.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

**Comment by pshipton** [comment]
Updated to address comments.

**Comment by pshipton** [comment]
Tweaked some other javadoc.

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
I'm satisfied that the recent changes don't invalidate the successful build at https://openj9-jenkins.osuosl.org/job/PullRequest-OpenJ9/2175/; I also did a local build to confirm the UMA changes are good.

**Pull request merged**